src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -395,9 +395,14 @@ uint32_t PolicyHandler::GetAppIdForSending() const {
           (*first)->device(),
           application_manager_.connection_handler().get_session_observer());
 
-      if (kDeviceAllowed ==
-          policy_manager_->GetUserConsentForDevice(
-              device_params.device_mac_address)) {
+      const bool device_allowed = kDeviceAllowed ==
+                                  policy_manager_->GetUserConsentForDevice(
+                                      device_params.device_mac_address);
+      const bool app_in_none =
+          application_manager_.application(app_id)->hmi_level() ==
+          mobile_apis::HMILevel::HMI_NONE;
+
+      if (device_allowed && !app_in_none) {
```

Comment:
@Kozoriz 
That condition won't cover requirement "app_4 (which in NONE) doesn't take part in PTU (except of the case when app_4 is the only application being run on SDL)"